src/packages/@workspaces/features/collection-explorer/layout/CollectionExplorer.svelte

```diff
@@ -447,7 +447,9 @@
     border-radius: 0%;
   }
   textarea {
-    outline: none;
+    outline-color: var(--sparrow-blue);
+    border: none;
+    border-radius: 8px !important;
   }
```

Comment:
we have stopped using this outdated color codes, lets follow the latest patterns